incubator/hnc/config/crd/bases/hnc.x-k8s.io_hierarchyconfigurations.yaml

```diff
@@ -95,10 +95,9 @@ spec:
                       would cause a cycle) \n - \"CritAncestor\": a critical error
                       exists in an ancestor namespace, so this namespace is no longer
                       being updated either. \n - \"HNSMissing\": this namespace is
-                      an owned namespace (created by reconciling an hns instance),
-                      but the hns instance is missing in its owner (referenced in
-                      its owner annotation) or the owner namespace is missing. \n
-                      - \"CannotPropagateObject\": this namespace contains an object
+                      a subnamespace, but the hierarchicalnamespace instance referenced
+                      in its `ownerReference` annotation does not exist in the parent.
```

Comment:
Didn't you change this to `subnamespaceOf` above?

Reply:
Good catch, fixed.